bloggies_backend/src/routes/posts.ts

```diff
@@ -12,9 +12,15 @@ export const postsRouter = express.Router();
 postsRouter.post("/", ensureLoggedIn, async function (req: Request, res: Response, next: NextFunction) {
   try {
     const { user_id } = req.user;
-    const { title, description, body, is_premium } = req.body;
-    const post = await Post.createPost(title, description, body, user_id, is_premium);
-    return res.status(201).send({ post });
+    const user = await User.getUser(user_id);
+    
+    if (user.membership_status === "active") {
```

Comment:
Will you will change the active to the 'ACTIVE' variable you are exporting from membershipStatus.ts once the two branches are merged to main?

Reply:
Yup! :D